tensorboard/webapp/tb_wrapper/tb_wrapper_component.ts

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+
+@Component({
+  selector: 'tensorboard-wrapper-component',
+  template: `
+    <app-header></app-header>
```

Comment:
Hmm, I don't think this is a change we want. It makes app-header customization near impossible. Can the header live outside of the router-outlet? (also, having `<header>` inside `<main>` sounds off.

Reply:
Done in the diffbase PR.

Right, we can expect the header to hide/show any route-specific UI it owns, like its 'plugin-selector'.